gitlab/v4/objects/pipelines.py

```diff
@@ -183,8 +183,15 @@ class ProjectPipelineScheduleVariableManager(
     _update_attrs = RequiredOptional(required=("key", "value"))
 
 
+class ProjectPipelineSchedulePipelinesManager(ListMixin, RESTManager):
+    _path = "/projects/{project_id}/pipeline_schedules/{pipeline_schedule_id}/pipelines"
+    _obj_cls = ProjectPipeline
+    _from_parent_attrs = {"project_id": "project_id", "pipeline_schedule_id": "encoded_id"}
+
+
 class ProjectPipelineSchedule(SaveMixin, ObjectDeleteMixin, RESTObject):
     variables: ProjectPipelineScheduleVariableManager
+    pipelines: ProjectPipelineSchedulePipelinesManager
```

Comment:
We can use the singular for the managers here I think. Also `id` should be enough, the encoding is done elsewhere:

```suggestion
class ProjectPipelineSchedulePipelineManager(ListMixin, RESTManager):
    _path = "/projects/{project_id}/pipeline_schedules/{pipeline_schedule_id}/pipelines"
    _obj_cls = ProjectPipeline
    _from_parent_attrs = {"project_id": "project_id", "pipeline_schedule_id": "id"}


class ProjectPipelineSchedule(SaveMixin, ObjectDeleteMixin, RESTObject):
    variables: ProjectPipelineScheduleVariableManager
    pipelines: ProjectPipelineSchedulePipelineManager
```

Reply:
Done in 202bbdd0da5baf138d7e794451d49eccb623559f.